docs/basics/index.md

```diff
@@ -19,11 +19,11 @@ Data modeling is the process of creating a data model
 
 ### Virtual DOM
 
-Vue accomplishes this by building a virtual DOM to keep track of the changes it needs to make to the real DOM
+Vue.js accomplishes this by building a virtual DOM to keep track of the changes it needs to make to the real DOM
 
 ### Flux Design Pattern
 
-The components in Flux's architecture interact more like an EventBus and less like an MVC. As mentioned earlier, Flux is not actually a library or a framework, it is a new kind of architecture that Facebook created to work with React:
+The components in Flux's architecture interact more like an EventBus and less like an MVC. Flux is not actually a library or a framework, it is a new kind of architecture that Facebook created to work with React:
```

Comment:
I suggest to change this:

"it is a new kind of architecture that Facebook created to work with React"

for this

"it's a pattern created by Facebook and implemented in React"